ljh/ljh.go

```diff
@@ -52,13 +53,21 @@ type Writer struct {
 	ChanNum         int
 	Presamples      int
 	Samples         int
+	FramesPerSample int
 	Timebase        float64
-	TimestampOffset float64
```

Comment:
I thought we were going to drop this? Maybe just rename it something like "time writing started"